### ISSUE TITLE
Using JobScheduler.Hangfire module, XAF Blazor application crash when opening Workers tab of Hangire Job

### ISSUE THREAD

**Issue by ss-carlo** [eXpandFramework/eXpand#978]
Hi,

While in the details view of the Hangfire Job, when I try to open the Workers tab, the application suddenly crashes with the following error message:

`The error occurred:

	Type:       NullReferenceException
	Message:    Object reference not set to an instance of an object.
	Data:       0 entries
	Stack trace:

   at Xpand.XAF.Modules.Reactive.Services.AttributesExtensions.<>c.<ReadOnlyCollection>b__2_4(ValueTuple`2 t)
   at System.Linq.EnumerableEx.DoCore[TSource](IEnumerable`1 source, Action`1 onNext, Action`1 onError, Action onCompleted)+MoveNext() in /_/Ix.NET/Source/System.Interactive/System/Linq/Operators/Do.cs:line 129
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Reactive.Linq.ObservableImpl.SelectMany`2.EnumerableSelector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/SelectMany.cs:line 1361
	InnerException is null`

Sample video:
https://user-images.githubusercontent.com/55209461/217061893-595240bd-0760-45ec-ad51-8da3930ea95c.mp4

**Comment by apobekiaris** [comment]
post your sample

**Comment by ss-carlo** [comment]
Here is the sample project

[Sample](https://drive.google.com/file/d/1s4QQUYwF3BLVJKpmOrwgQtHoAFTlBmiK/view?usp=sharing)

**Comment by apobekiaris** [comment]
The pre-release [4.222.2.0](https://github.com/eXpandFramework/Reactive.XAF/releases/tag/4.222.2.0) in the [Reactive.XAF](https://github.com/eXpandFramework/Reactive.XAF/tree/lab) `lab` branch  includes commits that relate to this task:

* [Fix JobScheduler JobWaroker UI exceptions #978](https://github.com/eXpandFramework/Reactive.XAF/commit/67471239bdcb3a7a88a57b57cae6f7b1378279a5)
* [XpandModelEditor Fails for Blazor and JobScheduler  #978](https://github.com/eXpandFramework/Reactive.XAF/commit/079c465fa09dc439ecf11e3f490405cc1389a928)
* [Update TestApplication Blazor Startup and JobScheduler #978](https://github.com/eXpandFramework/Reactive.XAF/commit/9586e7451451dc28f30cc13dde42122da61c04d7)

To minimize version conflicts we recommend that you use the [Xpand.XAF.Core.All](https://www.nuget.org/packages/Xpand.XAF.Core.All), [Xpand.XAF.Win.All](https://www.nuget.org/packages/Xpand.XAF.Win.All), [Xpand.XAF.Web.All](https://www.nuget.org/packages/Xpand.XAF.Web.All) packages. Doing so, all packages will be at your disposal and .NET will add a dependecy only to those packages that you actually use and not to all (see the [Modules installation-registrations](https://youtu.be/LvxQ-U_0Sbg) youtube video).
<details>
    <summary>Released packages:</summary>
<a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions'>Xpand.Extensions v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Blazor'>Xpand.Extensions.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Mono.Cecil'>Xpand.Extensions.Mono.Cecil v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud'>Xpand.Extensions.Office.Cloud v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud.Google.Blazor'>Xpand.Extensions.Office.Cloud.Google.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Reactive'>Xpand.Extensions.Reactive v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF'>Xpand.Extensions.XAF v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF.Xpo'>Xpand.Extensions.XAF.Xpo v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Common v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.EasyTest v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/tools/Xpand.VersionConverter'>Xpand.VersionConverter v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Core.All v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/AutoCommit'>Xpand.XAF.Modules.AutoCommit v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Blazor'>Xpand.XAF.Modules.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/BulkObjectUpdate'>Xpand.XAF.Modules.BulkObjectUpdate v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneMemberValue'>Xpand.XAF.Modules.CloneMemberValue v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneModelView'>Xpand.XAF.Modules.CloneModelView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Email'>Xpand.XAF.Modules.Email v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/GridListEditor'>Xpand.XAF.Modules.GridListEditor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/HideToolBar'>Xpand.XAF.Modules.HideToolBar v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Hangfire'>Xpand.XAF.Modules.JobScheduler.Hangfire v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Notification'>Xpand.XAF.Modules.JobScheduler.Notification v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/MasterDetail'>Xpand.XAF.Modules.MasterDetail v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelMapper'>Xpand.XAF.Modules.ModelMapper v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelViewInheritance'>Xpand.XAF.Modules.ModelViewInheritance v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google'>Xpand.XAF.Modules.Office.Cloud.Google v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Calendar'>Xpand.XAF.Modules.Office.Cloud.Google.Calendar v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Tasks'>Xpand.XAF.Modules.Office.Cloud.Google.Tasks v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.DocumentStyleManager'>Xpand.XAF.Modules.Office.DocumentStyleManager v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/OneView'>Xpand.XAF.Modules.OneView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/PositionInListView'>Xpand.XAF.Modules.PositionInListView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ProgressBarViewItem'>Xpand.XAF.Modules.ProgressBarViewItem v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RazorView'>Xpand.XAF.Modules.RazorView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive'>Xpand.XAF.Modules.Reactive v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger'>Xpand.XAF.Modules.Reactive.Logger v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Client.Win'>Xpand.XAF.Modules.Reactive.Logger.Client.Win v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Hub'>Xpand.XAF.Modules.Reactive.Logger.Hub v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Rest'>Xpand.XAF.Modules.Reactive.Rest v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RefreshView'>Xpand.XAF.Modules.RefreshView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SequenceGenerator'>Xpand.XAF.Modules.SequenceGenerator v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Speech'>Xpand.XAF.Modules.Speech v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SpellChecker'>Xpand.XAF.Modules.SpellChecker v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/StoreToDisk'>Xpand.XAF.Modules.StoreToDisk v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SuppressConfirmation'>Xpand.XAF.Modules.SuppressConfirmation v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/TenantManager'>Xpand.XAF.Modules.TenantManager v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewEditMode'>Xpand.XAF.Modules.ViewEditMode v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewItemValue'>Xpand.XAF.Modules.ViewItemValue v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewWizard'>Xpand.XAF.Modules.ViewWizard v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Windows'>Xpand.XAF.Modules.Windows v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Web.All v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Win.All v.4.222.2</a></br>
</details>

Please update the related Nuget packages and test if issues is addressed. These are nightly nuget packages available only from our [NugetServer](https://xpandnugetserver.azurewebsites.net/nuget/).

If you do not use these packages directly but through a module of the main eXpandFramework project, please wait for the bot to notify you again when integration is finished or update the related packages manually.

Thanks a lot for your contribution.

**Comment by ss-carlo** [comment]
Hi @apobekiaris,

I have updated my DevExpress version to 22.2.4 to be able to use the pre-release Nuget packages. I also used the [Xpand.XAF.Core.All](https://www.nuget.org/packages/Xpand.XAF.Core.All) and [Xpand.XAF.Web.All](https://www.nuget.org/packages/Xpand.XAF.Web.All) as recommended but it still did not fix the issue. It actually made the Blazor application crash when I try to navigate into the JobScheduler menu [Sample Video](https://drive.google.com/file/d/1phDL21qg-uqf5OfEnB0LcuHzqjyGvY7i/view?usp=sharing) 

Here is the new error message:
`The error occurred:

	Type:       ArgumentNullException
	Message:    Value cannot be null. (Parameter 'context')
	Data:       0 entries
	Stack trace:

   at System.Reactive.Linq.Observable.ObserveOn[TSource](IObservable`1 source, SynchronizationContext context) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable.Concurrency.cs:line 62
   at Xpand.XAF.Modules.Reactive.Services.XafApplicationRxExtensions.WhenWindowCreated(XafApplication application, Boolean isMain, Boolean emitIfMainExists)
   at Xpand.XAF.Modules.Reactive.Services.XafApplicationRxExtensions.WhenSynchronizationContext(XafApplication application)
   at Xpand.XAF.Modules.Reactive.Services.ShowMessageExtensions.ShowMessages(XafApplication application)
   at Xpand.XAF.Modules.Reactive.RxApp.Connect(XafApplication application)
   at Xpand.XAF.Modules.Reactive.RxApp.<>c__DisplayClass12_0.<Connect>b__0(XafApplication application)
   at System.Reactive.Linq.ObservableImpl.Defer`1._.Run() in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Defer.cs:line 37
	InnerException is null`

Thanks

**Comment by apobekiaris** [comment]
Using the `next`  minor I did:
1. New Wizard Blazor solution
2. Add the next in the blazor project
   ```xml
   <PackageReference Include="Xpand.XAF.Modules.JobScheduler.Hangfire" Version="4.222.2.1" />
    <PackageReference Include="Hangfire.MemoryStorage" Version="1.7.0" />
   ```
3. Added the next in the `BlazorModule.cs`
```cs

[assembly: HostingStartup(typeof(HostingStartup))]
[assembly: HostingStartup(typeof(HangfireStartup))]
[assembly: HostingStartup(typeof(Xpand.XAF.Modules.Blazor.BlazorStartup))]
namespace DXApplication26.Blazor.Server;

[JobProvider]
public class Job {
	public IServiceProvider ServiceProvider { get; }
	public Job() { }
	[ActivatorUtilitiesConstructor]
	public Job(IServiceProvider provider) => ServiceProvider = provider;

	[JobProvider]
	public async Task<bool> ImportOrders() 
		=> await ServiceProvider.RunWithStorageAsync(application => Observable.Using(
			() => application.CreateNonSecuredObjectSpace(typeof(Order)),objectSpace 
				=> Observable.Range(0, 10).Do(i => {
					var order = objectSpace.CreateObject<Order>();
					order.OrderID = i;
				}).Finally(objectSpace.CommitChanges)).To(true));

	public void Failed() {
		throw new NotImplementedException();
	}


}

public class Order:BaseObject {
	public Order(Session session) : base(session) {
	}

	int _orderID;

	public int OrderID {
		get => _orderID;
		set => SetPropertyValue(nameof(OrderID), ref _orderID, value);
	}
}   
```
4. In the same file
```cs
public DXApplication26BlazorModule() {
       RequiredModuleTypes.Add(typeof(JobSchedulerModule));
    }
    public override void AddGeneratorUpdaters(ModelNodesGeneratorUpdaters updaters) {
	    base.AddGeneratorUpdaters(updaters);
	    updaters.Add(new JoSchedulerSourceUpdater());
    }
```
5. Setup a Hangfire storage in Startup.cs
```cs
GlobalConfiguration.Configuration.UseMemoryStorage();
```

Doing so I was able to create and trigger a new job and open the Workers tab.

**Comment by apobekiaris** [comment]
The pre-release [4.222.2.0](https://github.com/eXpandFramework/Reactive.XAF/releases/tag/4.222.2.0) in the [Reactive.XAF](https://github.com/eXpandFramework/Reactive.XAF/tree/lab) `lab` branch  includes commits that relate to this task:

* [Fix wrong namespaces #978 #982](https://github.com/eXpandFramework/Reactive.XAF/commit/7dc1e463680c3e269013f27a1fa66ba72204e4b0)

To minimize version conflicts we recommend that you use the [Xpand.XAF.Core.All](https://www.nuget.org/packages/Xpand.XAF.Core.All), [Xpand.XAF.Win.All](https://www.nuget.org/packages/Xpand.XAF.Win.All), [Xpand.XAF.Web.All](https://www.nuget.org/packages/Xpand.XAF.Web.All) packages. Doing so, all packages will be at your disposal and .NET will add a dependecy only to those packages that you actually use and not to all (see the [Modules installation-registrations](https://youtu.be/LvxQ-U_0Sbg) youtube video).
<details>
    <summary>Released packages:</summary>
<a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions'>Xpand.Extensions v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Blazor'>Xpand.Extensions.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Mono.Cecil'>Xpand.Extensions.Mono.Cecil v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud'>Xpand.Extensions.Office.Cloud v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud.Google.Blazor'>Xpand.Extensions.Office.Cloud.Google.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Reactive'>Xpand.Extensions.Reactive v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF'>Xpand.Extensions.XAF v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF.Xpo'>Xpand.Extensions.XAF.Xpo v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Common v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.EasyTest v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/tools/Xpand.VersionConverter'>Xpand.VersionConverter v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Core.All v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/AutoCommit'>Xpand.XAF.Modules.AutoCommit v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Blazor'>Xpand.XAF.Modules.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/BulkObjectUpdate'>Xpand.XAF.Modules.BulkObjectUpdate v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneMemberValue'>Xpand.XAF.Modules.CloneMemberValue v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneModelView'>Xpand.XAF.Modules.CloneModelView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Email'>Xpand.XAF.Modules.Email v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/GridListEditor'>Xpand.XAF.Modules.GridListEditor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/HideToolBar'>Xpand.XAF.Modules.HideToolBar v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Hangfire'>Xpand.XAF.Modules.JobScheduler.Hangfire v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Notification'>Xpand.XAF.Modules.JobScheduler.Notification v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/MasterDetail'>Xpand.XAF.Modules.MasterDetail v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelMapper'>Xpand.XAF.Modules.ModelMapper v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelViewInheritance'>Xpand.XAF.Modules.ModelViewInheritance v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google'>Xpand.XAF.Modules.Office.Cloud.Google v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Calendar'>Xpand.XAF.Modules.Office.Cloud.Google.Calendar v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Tasks'>Xpand.XAF.Modules.Office.Cloud.Google.Tasks v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.DocumentStyleManager'>Xpand.XAF.Modules.Office.DocumentStyleManager v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/OneView'>Xpand.XAF.Modules.OneView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/PositionInListView'>Xpand.XAF.Modules.PositionInListView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ProgressBarViewItem'>Xpand.XAF.Modules.ProgressBarViewItem v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RazorView'>Xpand.XAF.Modules.RazorView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive'>Xpand.XAF.Modules.Reactive v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger'>Xpand.XAF.Modules.Reactive.Logger v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Client.Win'>Xpand.XAF.Modules.Reactive.Logger.Client.Win v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Hub'>Xpand.XAF.Modules.Reactive.Logger.Hub v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Rest'>Xpand.XAF.Modules.Reactive.Rest v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RefreshView'>Xpand.XAF.Modules.RefreshView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SequenceGenerator'>Xpand.XAF.Modules.SequenceGenerator v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Speech'>Xpand.XAF.Modules.Speech v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SpellChecker'>Xpand.XAF.Modules.SpellChecker v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/StoreToDisk'>Xpand.XAF.Modules.StoreToDisk v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SuppressConfirmation'>Xpand.XAF.Modules.SuppressConfirmation v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/TenantManager'>Xpand.XAF.Modules.TenantManager v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewEditMode'>Xpand.XAF.Modules.ViewEditMode v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewItemValue'>Xpand.XAF.Modules.ViewItemValue v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewWizard'>Xpand.XAF.Modules.ViewWizard v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Windows'>Xpand.XAF.Modules.Windows v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Web.All v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Win.All v.4.222.2</a></br>
</details>

Please update the related Nuget packages and test if issues is addressed. These are nightly nuget packages available only from our [NugetServer](https://xpandnugetserver.azurewebsites.net/nuget/).

If you do not use these packages directly but through a module of the main eXpandFramework project, please wait for the bot to notify you again when integration is finished or update the related packages manually.

Thanks a lot for your contribution.

**Comment by ss-carlo** [comment]
> Using the `next` minor I did:
> 
> 1. New Wizard Blazor solution
> 2. Add the next in the blazor project
>    ```
>    <PackageReference Include="Xpand.XAF.Modules.JobScheduler.Hangfire" Version="4.222.2.1" />
>     <PackageReference Include="Hangfire.MemoryStorage" Version="1.7.0" />
>    ```
> 3. Added the next in the `BlazorModule.cs`
> 
> ```cs
> [assembly: HostingStartup(typeof(HostingStartup))]
> [assembly: HostingStartup(typeof(HangfireStartup))]
> [assembly: HostingStartup(typeof(Xpand.XAF.Modules.Blazor.BlazorStartup))]
> namespace DXApplication26.Blazor.Server;
> 
> [JobProvider]
> public class Job {
> 	public IServiceProvider ServiceProvider { get; }
> 	public Job() { }
> 	[ActivatorUtilitiesConstructor]
> 	public Job(IServiceProvider provider) => ServiceProvider = provider;
> 
> 	[JobProvider]
> 	public async Task<bool> ImportOrders() 
> 		=> await ServiceProvider.RunWithStorageAsync(application => Observable.Using(
> 			() => application.CreateNonSecuredObjectSpace(typeof(Order)),objectSpace 
> 				=> Observable.Range(0, 10).Do(i => {
> 					var order = objectSpace.CreateObject<Order>();
> 					order.OrderID = i;
> 				}).Finally(objectSpace.CommitChanges)).To(true));
> 
> 	public void Failed() {
> 		throw new NotImplementedException();
> 	}
> 
> 
> }
> 
> public class Order:BaseObject {
> 	public Order(Session session) : base(session) {
> 	}
> 
> 	int _orderID;
> 
> 	public int OrderID {
> 		get => _orderID;
> 		set => SetPropertyValue(nameof(OrderID), ref _orderID, value);
> 	}
> }   
> ```
> 
> 4. In the same file
> 
> ```cs
> public DXApplication26BlazorModule() {
>        RequiredModuleTypes.Add(typeof(JobSchedulerModule));
>     }
>     public override void AddGeneratorUpdaters(ModelNodesGeneratorUpdaters updaters) {
> 	    base.AddGeneratorUpdaters(updaters);
> 	    updaters.Add(new JoSchedulerSourceUpdater());
>     }
> ```
> 
> 5. Setup a Hangfire storage in Startup.cs
> 
> ```cs
> GlobalConfiguration.Configuration.UseMemoryStorage();
> ```
> 
> Doing so I was able to create and trigger a new job and open the Workers tab.

Thanks, I can confirm that this minor version fixes the issue!
